assets/src/scripts/monitoring-location/components/hydrograph/discrete-data.js

```diff
@@ -11,10 +11,13 @@ const GW_LEVEL_CLASS = 'gw-level-point';
  * @param {D3 scale} xScale
  * @param {D3 scale } yScale
  */
-export const drawGroundwaterLevels = function(svg, {levels, xScale, yScale}) {
+export const drawGroundwaterLevels = function(svg, {levels, xScale, yScale, enableClip}) {
```

Comment:
I did need to add a clip to drawing groundwater levels. We could remove this if the selector only gave us visible points for the graph but I figured there would be unexpected results so just but in the clip.